packages/controls/src/widget_bool.ts

```diff
@@ -79,10 +79,19 @@ export class CheckboxView extends DescriptionView {
       return;
     }
     const description = this.model.get('description');
-    this.descriptionSpan.innerHTML = description;
+    const plaintext_description = this.model.widget_manager.plaintext_sanitize(
```

Comment:
I think I disagree with using a plaintext_sanitize for situations like this- if HTML is disallowed then the text should be treated as plaintext, meaning strings containing `<>&` etc are rendered as-is as opposed to being stripped of all potential HTML elements (if I'm reading the code correctly).

Reply:
I agree - not allowing html should just use textContent, effectively just escaping any html it sees. I've pushed a commit implementing that approach.